### PR TITLE
Add simple library to print FPS on screen

### PR DESCRIPTION
**Pull request by Deryugin**

### CODE REVIEW

File: platform/mesa/cmds/osdemo_fb/osdemo_fb.c [hunk 1]
```diff
@@ -63,38 +64,19 @@ void init_buffers(void) {
...
-			time_prev = time;
-			fps = counter > 99 ? 99 : counter;
-			counter = 0;
-		}
-
-		fps_str[4] = '0' + fps / 10;
-		fps_str[5] = '0' + fps % 10;
-		fb_overlay_put_string(0, 0, "Embox MESA demo v0.2");
-		fb_overlay_put_string(0, 1, fps_str);
+		fps_set_format("Embox MESA demo v0.2\nFPS=%2d");
```
Comment (alexkalmuk) [inline platform/mesa/cmds/osdemo_fb/osdemo_fb.c]: I think an argument for `%2d` is expected?

File: src/lib/fps/fps.c [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+	assert(fb);
+
+	memset(msg_buf, 0, sizeof(msg_buf));
+
+	clock_gettime(CLOCK_REALTIME, &time);
+
+	counter++;
+
+	if (time.tv_sec != time_prev.tv_sec) {
```
Comment (alexkalmuk) [inline src/lib/fps/fps.c]: Shouldn't we check only one second expired (I mean exactly one, not 2 or 3...)?

File: platform/mesa/cmds/osdemo_fb/osdemo_fb.c (hunk 1 above)
Comment (Deryugin) [reply platform/mesa/cmds/osdemo_fb/osdemo_fb.c]: Fixed this anyway :)

File: src/lib/fps/fps.c (hunk 2 above)
Comment (Deryugin) [reply src/lib/fps/fps.c]: Fixed

**Pull request merged**